src/Pages/AboutUs.jsx

```diff
@@ -12,7 +12,7 @@ const AboutUs = () => {
     return (
         <>
             <Values />
-            <Quote quoteContent={aboutQuoteContent} quoteAuthor={aboutQuoteAuthor} />
```

Comment:
why is this needed?

Reply:
the quote banner not fits in to entire screen or same css not working for both..so passing this custom class to differentiate